src/Microsoft.VisualStudio.Services.Agent/Constants.cs

```diff
@@ -101,6 +105,12 @@ public static class ReturnCode
                 public const int RetryableError = 2;
                 public const int AgentUpdating = 3;
             }
+
+            public static class AgentConfigurationProvider
+            {
+                public static readonly string BuildReleasesAgentConfiguration = "BuildReleasesAgentConfiguration";
```

Comment:
taken care of enum to string and changing name to "BuildReleasesAgentConfiguration"